qa/backwards-5.0/build.gradle

```diff
@@ -16,9 +16,9 @@ apply plugin: 'elasticsearch.rest-test'
 integTest {
   includePackaged = true
   cluster {
-    numNodes = 2
-    numBwcNodes = 1
-    bwcVersion = "6.0.0-alpha1-SNAPSHOT"
+    numNodes = 4
+    numBwcNodes = 2
```

Comment:
I wonder if this is a bit too much to just run the new IndexingIT. The YAML tests will then also execute with that many nodes and the Windows VMs will take forever. It might also double the heap size needed to run the tests (4 * 2GB per ES instance?).